scripts/shared/hostname/install

```diff
@@ -0,0 +1,36 @@
+#!/bin/bash
+
+MESH_NAME=$(confget -f /etc/mesh.conf -s general "mesh-name")
+
+if [ -f '/etc/cjdroute.conf' ]; then
+    # Define new hostname
+    if [ -z "${NEWHOSTNAME}" ]; then
+    NEWHOSTNAME=$(sudo grep -m 1 '"ipv6"' /etc/cjdroute.conf | awk '{ print $2 }' | sed 's/[",]//g' | sed "s/.*:/$MESH_NAME-/g")
+    fi
+fi
+
+if ! [ -z "$(ifconfig  | grep tun0:)" ]; then
+    # Define new hostname
+    if [ -z "${NEWHOSTNAME}" ]; then
+    NEWHOSTNAME=$(sudo ifconfig ygg0 | grep -E 'inet6 2[0-9a-fA-F]{2}:' | awk '{print $2}' | sed 's/[",]//g' | sed "s/.*:/$MESH_NAME-Y/g")
+    fi
+fi
+
+# Set node name
+if ! [ -z "${NEWHOSTNAME}" ]; then
+
+    # Rename node based on cjdns address
+    oldhostname=$(hostname)
+    sudo echo "$NEWHOSTNAME" | sudo tee /etc/hostname > /dev/null
```

Comment:
Rename the device already, and replace the hostname in the hostname file.
```suggestion
    hostname "$NEWHOSTNAME"
    sudo sed -i -e "s/$oldhostname/$NEWHOSTNAME/" /etc/hostname
```

Reply:
`sudo echo "$NEWHOSTNAME" | sudo tee /etc/hostname > /dev/null`
will overwrite the content vs trying to do a find and replace.

Admittedly the first `sudo` in my line is not required

Reply:
Oh okay I see, sed is not needed yeah. But the hostname command is still good here I think.